packages/p2p/src/components/dp2p-blocked/__tests__/dp2p-blocked.spec.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Dp2pBlocked from '../dp2p-blocked';
+
+describe('<Dp2pBlocked />', () => {
+    it('it should render <Dp2pBlocked /> component', () => {
+        render(<Dp2pBlocked />);
+        expect(screen.getByTestId('dt_dp2p-blocked-container')).toBeInTheDocument();
```

Comment:
Base on the documentation the priority of the queries is like this:

`Queries Accessible to Everyone:`

- **getByRole**
- **getByLabelText**
- **getByPlaceholderText**
- **getByText**
- **getByDisplayValue**

`Semantic Queries`

- getByAltText
- getByTitle
- Test IDs

Reply:
Here instead of using the test Id you can check with role or text